rolf/io/data.py

```diff
@@ -270,14 +309,19 @@ def create_torch_datasets(
             train["label"].to_numpy(),
             train["filepath"].to_numpy(),
             img_dir=img_dir,
+            transform=self.train_transformer
```

Comment:
This won't allow us to parse a custom transform to the `create_torch_datasets` method